pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -149,10 +155,10 @@ func init() {
 	}
 
 	err = InfraSAs.addServiceAccount(
-		InfraDeploymentControllerServiceAccountName,
+		InfraDeploymentConfigControllerServiceAccountName,
 		authorizationapi.ClusterRole{
 			ObjectMeta: kapi.ObjectMeta{
-				Name: DeploymentControllerRoleName,
+				Name: DeploymentConfigControllerRoleName,
```

Comment:
This is not the deployment config controller but our custom replication controller controller